R/RsquareAdj.R

```diff
@@ -35,11 +35,12 @@
 
 ## cca result: no RsquareAdj
 RsquareAdj.cca <-
-    function(x, ...)
+    function (x, nperm = 1000) 
 {
-    R2 <- x$CCA$tot.chi/x$tot.chi
-    radj <- NA
-    list(r.squared = R2, adj.r.squared = radj)
+    r2 <- x$CCA$tot.chi / x$tot.chi
+    p <- permutest(x, nperm)
```

Comment:
How about passing `...` to `permutest()` here? That way people can call this with `parallel` or other functions should they need to speed this up.